src/pages/admin/reports/components/program-expenses-section/components/common/add-program-expense-record-modal/AddProgramExpenseRecordModal.module.scss

```
@@ -132,14 +132,31 @@
         background: c.$white;
         filter: drop-shadow(0 1px 2px rgba(c.$bluish-black, 0.05));
     }
+
+    :global(.char-limit-input--error) {
```

Stylelint (17.11.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)